torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -634,16 +634,27 @@ def rewrite_node(node):
                     return g.call_function(ttnn.reshape, args=(args[0], output_size))
                 return None
 
-            if node.target == torch.ops.aten.transpose.int:
-                dim0 = args[1]
-                dim1 = args[2]
+            if node.target in [torch.ops.aten.transpose.int, torch.ops.aten.t.default]:
                 output_size = node.meta["val"].size()
                 rank = len(output_size)
+                if node.target == torch.ops.aten.t.default:
```

Comment:
what's the difference between `torch.ops.aten.transpose` and `ops.aten.t`?

Reply:
I guess there are historical reasons but `aten.t` only transposes 2D tensor and `aten.transpose` can swap any two dims on n-D tensor

https://pytorch.org/docs/stable/generated/torch.t.html
https://pytorch.org/docs/stable/generated/torch.transpose.html

Reply:
noted, thank you